examples/block-explorer/explorer-index/Cargo.toml

```diff
@@ -11,9 +11,10 @@ crate-type = ['cdylib']
 fuel-indexer-macros = { version = "0.3", path = "../../../packages/fuel-indexer-macros", default-features = false }
 fuel-indexer-plugin = { version = "0.3", path = "../../../packages/fuel-indexer-plugin" }
 fuel-indexer-schema = { version = "0.3", path = "../../../packages/fuel-indexer-schema", default-features = false }
-fuel-tx = "0.23"
-fuels-core = "0.30"
-fuels-types = "0.30"
+fuel-tx = "0.26"
+fuels-core = { git = "https://github.com/FuelLabs/fuels-rs", branch = "segfault_magnet/wasm_friendly_abigen", default-features = false }
```

Comment:
@deekerno  
- Do we have any idea how long we'll have to use this branch? 
- Is there some target date by which we can expect to be back on master?

cc @segfault-magnet

Reply:
I haven't received an answer about how long we'll need to use the branch. But I'll be monitoring it so that we can switch as soon as it goes into master, and then as soon as it's released.

Reply:
@deekerno Could we create a tracking issue for that?

Reply:
Done here: https://github.com/FuelLabs/fuel-indexer/issues/605